cli/kubectl-hyperkube

```diff
@@ -22,9 +55,21 @@ def cli(ctx, config):
     ctx.ensure_object(dict)
     ctx.obj['hyperkube_config'] = _load_config(config)
     ctx.obj['url'] = ctx.obj['hyperkube_config']['url']
+
+    u = urlparse(ctx.obj['url'])
+    ctx.obj['hostname'] = u.netloc
+
     ctx.obj['stage'] = ctx.obj['hyperkube_config']['stage']
-    headers = {'X-Api-Key': ctx.obj['hyperkube_config']['x_api_key']}
+    ctx.obj['region'] = ctx.obj['hyperkube_config'].get('region', "us-west-2")
```

Comment:
This default should be a class attribute or global. Maybe do the following?

At the top of the module:
```
DEFAULT_AWS_REGION = os.environ.get('DEFAULT_AWS_REGION', 'us-west-2')
```
```